doc/classes/Vector2.xml

```diff
@@ -190,6 +190,20 @@
 				Returns the vector with all components rounded down (towards negative infinity).
 			</description>
 		</method>
+		<method name="from_angle" qualifiers="static">
+			<return type="Vector2">
+			</return>
+			<argument index="0" name="angle" type="float">
+			</argument>
+			<description>
+				Creates a unit [Vector2] rotated to the given [code]angle[/code] in radians. This is equivalent of doing [code]Vector2(cos(angle), sin(angle))[/code] or [code]Vector2.RIGHT.rotated(angle)[/code].
+				[codeblock]
+				print(Vector2.from_angle(0)) # Prints Vector2(1, 0).
+				print(Vector2(1, 0).angle()) # Prints 0, which is the angle used above.
+				print(Vector2.from_angle(PI/2)) # Prints Vector2(0, 1).
```

Comment:
```suggestion
				print(Vector2.from_angle(TAU / 4)) # Prints Vector2(0, 1).
```

Reply:
Why `TAU`? In both cases this produces `Vector2(-0, 1)` from my testing. Also `PI / 2` and similar is more common to see in math in comparison.

Reply:
I like TAU more for angles because TAU / 4.0 is one fourth of a circle, which makes more sense to me than PI / 2.0 which is half of half a circle, although you're right it's just a preference thing. Also according to the [GDScript style guide](https://docs.godotengine.org/en/stable/getting_started/scripting/gdscript/gdscript_styleguide.html#numbers) these numbers should have a `.0` at the end.

Reply:
That's not what the GDScript style guide says (although I'm not opposed to this change). It says that floats should have `x.0` or `0.x` instead of just `x.` or `.x`. In this case, `TAU` is already a float, so in the expression `TAU / 4`, `4` is not technically a float, it's an integer which is implicitly converted to a float. According to the style guide, either `TAU / 4` or `TAU / 4.0` is appropriate. That said, `TAU / 4.0` is more explicit and likely better.

Reply:
PI is more recognizable, so I'm leaving it.